prow/plugins/lgtm/lgtm.go

```diff
@@ -314,12 +345,49 @@ func handlePullRequest(gc ghLabelClient, pe github.PullRequestEvent, log *logrus
 		return nil
 	}
 
-	// Don't bother checking if it has the label... it's a race, and we'll have
-	// to handle failure due to not being labeled anyway.
 	org := pe.PullRequest.Base.Repo.Owner.Login
 	repo := pe.PullRequest.Base.Repo.Name
 	number := pe.PullRequest.Number
 
+	// Check if we have LGTM label
+	hasLGTM := false
+	labels, err := gc.GetIssueLabels(org, repo, number)
+	if err != nil {
+		log.WithError(err).Errorf("Failed to get labels.")
+	}
+	hasLGTM = github.HasLabel(LGTMLabel, labels)
+
+	if hasLGTM == true {
```

Comment:
nit:` s/hasLGTM == true/hasLGTM/`

You can actually get rid of `hasLGTM` entirely and just put `github.HasLabel(LGTMLabel, labels)` here.